flutter/lib/resources/resource_manager.dart

```diff
@@ -23,6 +24,20 @@ class BatchPreset {
   });
 }
 
+enum ResourceTypeEnum { model, lib, datasetData, datasetGroundtruth }
+
+class Resource {
+  final String path;
+  final ResourceTypeEnum type;
```

Comment:
What is the purpose of this field?
I kinda like the idea of having it, it could maybe be useful, but right now we set it but never read it in the program.

Reply:
In the first implementation I use it to check if the resource is a model file and run checksum validation only against model files. But now I just run checksum validation against any files which have a valid `md5checksum`.
We can remove `ResourceTypeEnum type` but I also think it will be helpful to know the type of the resource.